tests/unit/test_airflow.py

```diff
@@ -757,3 +757,374 @@ def test_model_config_from_amazon_alg_estimator(sagemaker_session):
     }
 
     assert config == expected_config
+
+
+def test_transformer_config(sagemaker_session):
+    tf_transformer = transformer.Transformer(
+        model_name="tensorflow-model",
+        instance_count="{{ instance_count }}",
```

Comment:
This is Jinja templates. It will be evaluated during Airflow runtime. For example, I can put something in database (using xcom in ariflow) and do "{{ task_instance.xcom_pull(task_id='task', key='key') }}" to get the record in the table 'task' with key 'key'. Here in unit tests I just make some random Jinja templating strings for testing purpose.